ouroboros-consensus/src/Ouroboros/Consensus/Node/ErrorPolicy.hs

```diff
@@ -27,8 +33,8 @@ import           Ouroboros.Consensus.Util.ResourceRegistry
                      (RegistryClosedException, ResourceRegistryThreadException,
                      TempRegistryException)
 
-consensusErrorPolicy :: ErrorPolicies
-consensusErrorPolicy = ErrorPolicies {
+consensusErrorPolicy :: forall blk. (Typeable blk, StandardHash blk) => Proxy blk -> ErrorPolicies
```

Comment:
Am I correct in remembering that you said you don't mind lines longer than 80 characters?

Sorry to disappoint you, but we do try to stick to 80 characters, unless it is helpful in combination with alignment :grinning:.

When a type signature is too long, we use the following style:

```suggestion
consensusErrorPolicy ::
     forall blk. (Typeable blk, StandardHash blk)
  => Proxy blk
  -> ErrorPolicies
```

(When there are too many constraints, we wrap those too).